app/templates/partials/widgets/relationship_widget.html

```diff
@@ -0,0 +1,60 @@
+{% set label = {
+  'for': answer.id,
+  'text': answer.current_person + ' ' + answer.label + ' ' + answer.other_person
```

Comment:
Leaving for @hamishtaplin to figure out how to do this it's just to get get the selenium tests to pass